### ISSUE TITLE
Cached old eslintrc configuration

### ISSUE THREAD

**Issue by akobrina** [microsoft/vscode-eslint#1438]
I have installed ESlint for VS Code on a MAC OS. Everything was working as expected and highlighting the errors according to the config settings in my .eslintrc.js, which is located in the root folder of my project. But now, I after I have tweaked the config to remove "no-restricted-properties" setting and replaced it with "no-restricted-syntax" instead, I can still see the old rule loading with my custom message. And the new rule is not being picked up. I tried npm uninstalling eslint and uninstalling the eslint plugin for VS code, reloading the window, clearing everything and starting from scratch - nothing is helping. ESlint somehow still is picking up the old config setting. Please help.

**Comment by dbaeumer** [comment]
You can set `eslint.debug`  to `true`. This prints into the ESLint output channel from were rules and configs are resolved and which values are used. This should allow you to track down why it is not picked up.

You can alternatively run eslint in the terminal and see what is happening there for a comparison.

**Comment by akobrina** [comment]
Thank you for the fast reply @dbaeumer. I figured out what was wrong, so let me describe it here in case someone else will encounter it. Basically, I was implementing eslint for a new project and I had a test file going to check each new eslint rule. My test file looked something like this, with each code block prepended with a comment that I though would not be evaluated by eslint:
```
/* eslint no-restricted-properties: [2, { property: 'html' , message: 'HTML name is reserved because the compiler will create it.' }] */
var myObject = {
    main: function () {
        this.html;
    },
    html: {}
};
```
In reality, I learned that ESlint DOES see these comments as linting rules. I though ESlint only looks at the .eslintrc.* file for rules! As soon as I deleted the comment, I no longer see the problem and my proper rule from .eslintrc file is being picked up.

As a suggestion, I would recommend making the debugging of this easier, since adding the --debug mode did not let me know that the reason eslint is marking the code is because it is picking up the rule from the comment block.
